lib/modules/manager/cargo/types.ts

```diff
@@ -38,3 +38,30 @@ export interface CargoRegistries {
   // maps registry names to URLs
   [key: string]: string;
 }
+
```

Comment:
use zod schema instead, checkout latest poetry refactoring by @zharinov

Reply:
Moved the lockfile to `zod`. Ideally we'd move `Cargo.toml` to it as well, though I'd prefer to focus on `update-lockfile` in this MR.